db/seeds.rb

```diff
@@ -1,25 +1,28 @@
 def seed
-  add_admin
-  add_initial_order_types
-  add_initial_orders
+  if ENV['SEED_DB']
+    add_admin
+    add_initial_order_types
+    add_initial_orders
+  end
 end
 
 def add_admin
   User.connection.schema_cache.clear!
   User.reset_column_information
+  admin_email = ENV['ADMIN_EMAIL'] || 'user@example.com'
```

Comment:
Add variables for password and api token here

Reply:
Why?